src/components/register/steps/ticket/level.tsx

```diff
@@ -34,12 +35,18 @@ const AddonsContainer = styled.section`
 `
 
 const TicketLevel = (_: RouteComponentProps) => {
-	const { register, handleSubmit } = useForm<TicketLevelModel>()
+	const { register, watch, handleSubmit } = useForm<TicketLevelModel>()
 	const { ticketLevels, registrationExpirationDate } = useSiteMetadata()
 	const dispatch = useAppDispatch()
 
 	const expirationDate = DateTime.fromISO(registrationExpirationDate)
 
+	useEffect(() => {
+		const subscription = watch(data => dispatch(ChangeTicketLevel.create(data)))
+
+		return () => subscription.unsubscribe()
```

Comment:
Since we need to return a function, can we not just return `subscription.unsubscribe`?